src/agenteval/leaderboard/models.py

```diff
@@ -12,3 +18,83 @@ class LeaderboardSubmission(BaseModel):
 
     results: list[TaskResult] | None = None
     submission: SubmissionMetadata = Field(default_factory=SubmissionMetadata)
+
+
+@dataclass
+class Readme:
+    text_content: str
+    """Plain-text content section of the README."""
+    configs: dict[str, list[str]]
+    """Mapping of config names to the splits they define"""
+    features: Features
+    """
+    Features schema for leaderboard submission data
+    The dataset_features.yml file is the definitive definition of the schema.
+    All config names and splits must use the same schema.
```

Comment:
fwiw I'm not sure this is true -- I think different configs in the dataset could have different schema. Datsets.load_dataset is only ever trying to load one subset/config at a time.